tests/storage/hotplug.go

```diff
@@ -1576,8 +1598,13 @@ var _ = Describe(SIG("Hotplug", func() {
 			verifyWriteReadData(vmi, targets[0])
 			vmi, err = virtClient.VirtualMachineInstance(vmi.Namespace).Get(context.Background(), vmi.Name, metav1.GetOptions{})
 			Expect(err).ToNot(HaveOccurred())
-			By("deleting the attachment pod a few times, try to make the currently attach volume break")
-			for i := 0; i < 10; i++ {
+
+			By("deleting the attachment pod, try to make the currently attached volume break")
+			if limitHotplugPodCreation {
+				createPodLimitingResourceQuota(vmi.Namespace)
+				defer deleteQuota(vmi.Namespace)
```

Comment:
The clean way to do this kind of thing in ginkgo is DeferCleanup (imagine test exiting uncleanly)